applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -996,7 +989,7 @@ class TrilinosBlockBuilderAndSolver
     double mScaleFactor = 1.0;         /// The manually set scale factor
 
     /* Flags */
-    SCALING_DIAGONAL mScalingDiagonal; /// We identify the scaling considered for the dirichlet dofs
+    SCALING_DIAGONAL mScalingDiagonal = SCALING_DIAGONAL::NO_SCALING;; /// We identify the scaling considered for the dirichlet dofs
```

Comment:
Maybe we should do the same in the serial B&S

Reply:
@KratosMultiphysics/technical-committee recommends to change the `mScalingDiagonal` to the same as the one in the `GetDefaultParameters`.

Reply:
What do I do then @KratosMultiphysics/technical-committee ?

Reply:
Yes, but it was on purpose at the time. To avoid break things. Maybe if I do archaeology I can show the discussion